de-webapp/src/main/webapp/WEB-INF/jsp/bootstrapError.jsp

```diff
@@ -1,4 +1,5 @@
 <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ include file="react_include.jsp" %>
```

Comment:
This `react_include.jsp` file contains `<link>` and `<script>` tags that are intended to go inside the `<header>` tag of the HTML doc.

Notice the placement of this `include` in the `de.jsp` page is after most other `<script>` tags:
https://github.com/cyverse-de/ui/blob/a30f4c9914ee589a54731ce503a97b857bd7e10a/de-webapp/src/main/webapp/WEB-INF/jsp/de.jsp#L65-L68
Also in `belphegor.jsp`:
https://github.com/cyverse-de/ui/blob/295a657052f10fbcb6400c9e8f02151ff7201828/de-webapp/src/main/webapp/WEB-INF/jsp/belphegor.jsp#L10-L18

Reply:
I moved where I'm adding the include, but I also had to then copy over another line from the `de.jsp` and `belphegor.jsp` files:
```
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
```
Without that, I would get errors along the lines of "...not a supported stylesheet MIME type, and strict MIME checking is enabled."